.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+#### Ticket (For Vimeo staff only)
```

Comment:
As the first line of the template, do you think non-Vimeo Staff will see this and assume the entire template doesn't apply to them rather than just this section? I have gotten that feeling before when it's been included as the second line as it is within the Android public repos, like [this](https://github.com/vimeo/vimeo-networking-java/blob/v2.0.0/.github/PULL_REQUEST_TEMPLATE.md). I'm not sure what the best approach would be to solve this if it is a problem. Perhaps reordering the section down further? - But I like having the ticket number at the top. Perhaps better copy?